crates/bevy_pbr/src/meshlet/meshlet_bindings.wgsl

```diff
@@ -55,6 +55,7 @@ struct DrawIndirectArgs {
 @group(0) @binding(7) var<storage, read> meshlet_previous_occlusion: array<u32>; // 1 bit per cluster (instance of a meshlet), packed as a bitmask
 @group(0) @binding(8) var<uniform> view: View;
 @group(0) @binding(9) var depth_pyramid: texture_2d<f32>; // Generated from the first raster pass (unused in the first pass but still bound)
+var<push_constant> aabb_uv_to_viewport: vec2<f32>; // Multiplier to convert UV-space AABB to view-space (unused in the first pass but still bound)
```

Comment:
Those are guaranteed to be usable on all meshlet-capable platforms, right?

From looking at https://github.com/gpuweb/gpuweb/issues/75 I think that Metal doesn't quite support them, so wgpu emulates them I guess?

Reply:
No idea, but they work on all non-wasm platforms.

Reply:
OK, cool.